components/bootloader_support/src/bootloader_init.c

```diff
@@ -90,6 +90,10 @@ void bootloader_enable_random(void)
 
 void bootloader_print_banner(void)
 {
-    ESP_LOGI(TAG, "ESP-IDF %s 2nd stage bootloader", IDF_VER);
-    ESP_LOGI(TAG, "compile time " __TIME__);
+#ifdef CONFIG_MCUBOOT
+	ESP_LOGI(TAG, "## MCUBoot %s ESP32 bootloader", IDF_VER);
```

Comment:
Would be possible to use `printk` or Zephyr's logging subsystem instead?

Reply:
These kind of macros are mapped to MCUboot's logging functions. In my opinion we could keep as is.

Reply:
In fact that code is not even used, removing it.